src/main/java/org/broadinstitute/consent/http/models/DataAccessRequestData.java

```diff
@@ -555,6 +556,14 @@ public void setInternalCollaborators(
     this.internalCollaborators = internalCollaborators;
   }
 
+  public List<Collaborator> getLabAndInternalCollaborators() {
+   List<Collaborator> empty = Collections.emptyList();
+    return Stream.of(Objects.requireNonNullElse(labCollaborators, empty),
+            Objects.requireNonNullElse(internalCollaborators, empty))
+        .flatMap(List::stream)
+        .collect(Collectors.toList());
```

Comment:
I would use `List.of()` instead of `Collections.emptyList()`. Also `List.of()` doesn't allocate an object so there's no need to create a local variable for it. And `.toList()` is a better way to write `.collect(Collectors.toList())`.
```suggestion
    return Stream.of(Objects.requireNonNullElse(labCollaborators, List.of()),
            Objects.requireNonNullElse(internalCollaborators, List.of())
        .flatMap(List::stream)
        .toList();
```

Also these fields should be fixed so they're never `null`, but that might be a change for a future PR.

Reply:
I needed the type to be List<Collaborator> instead of List<?> which is why I created the variable, but if I just use the getters it resolves the type issue and the null case